<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="originalVideo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="changedVideo">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>270</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>10</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jaapokki subtract</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INSERT FILTER</string>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jaapokki subtract</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ORIGINAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jaapokki subtract</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CHANGED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jaapokki subtract</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAVE IMG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>10</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jaapokki subtract</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOAD IMG</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
